src/initStyles.js

```diff
@@ -2,7 +2,9 @@ export function initStyles(shadowDom, styleTargets) {
   const useGlobal = styleTargets.includes('global')
 
   const documentSheets = useGlobal
-    ? [...document.styleSheets]
+    ? [...document.styleSheets].filter(({href: styleHref}) =>
+        !styleHref || styleHref.startsWith(`${window.location.protocol}//${window.location.host}`)
```

Comment:
Be good to see if this works - `styleHref.includes(window.location.host)`. Just a cleaner approach.